### DISCUSSION TITLE
Add vibration on mobile platforms when hitting circles/sliders

### DISCUSSION THREAD

**Discussion by Vortex100K** [ppy/osu#17037] [Ideas]
I think vibrations would give the game some sort of depth especially for osu where the main thing your doing is tapping

it could also be cool if the vibrations where universal
instead of just happening in gameplay where an object is pressed for example in the song select or menu etc

**Comment by LOKSTED** [comment]
This would also help with the audio delay introduced by bluetooth earbuds. The tap sound is sent through bluetooth and gets delayed, while tap vibration could be felt instantly. This was implemented well in Hatsune Miku Colorful Stage

**Comment by AFasterSlowpoke** [comment]
it's a good idea, but i think there needs to be a toggle to turn off the vibrations if you want to.
